templates/repository/common/CONTRIBUTING.md

```diff
@@ -13,37 +13,38 @@ https://github.com/ory/meta/blob/master/templates/repository/common/CONTRIBUTING
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Introduction](#introduction)
-- [FAQ](#faq)
-- [How can I contribute?](#how-can-i-contribute)
-- [Communication](#communication)
-- [Contribute examples](#contribute-examples)
-- [Contribute code](#contribute-code)
-- [Contribute documentation](#contribute-documentation)
-- [Disclosing vulnerabilities](#disclosing-vulnerabilities)
-- [Code style](#code-style)
-  - [Working with forks](#working-with-forks)
-- [Conduct](#conduct)
+- [Contribute to Ory $PROJECT](#contribute-to-ory-project)
+  - [Introduction](#introduction)
+  - [FAQ](#faq)
+  - [How can I contribute?](#how-can-i-contribute)
+  - [Communication](#communication)
+  - [Contribute examples](#contribute-examples)
+  - [Contribute code](#contribute-code)
+  - [Contribute documentation](#contribute-documentation)
+  - [Disclosing vulnerabilities](#disclosing-vulnerabilities)
+  - [Code style](#code-style)
+    - [Working with forks](#working-with-forks)
+  - [Conduct](#conduct)
```

Comment:
This is still here. Please save the file without running Prettier by pressing `ctrl-alt-s` (on Linux or Windows). Not sure what the hotkey on macOS is. 

Alternatively, you can stage (select to be committed) only certain parts of a document:

- on the CLI: run `git commit -p`, then enter `y` if you want to commit the current part or `n` if you don't want to commit it
- in VSCode: enter "Source Control" view, double-click to see the diff of your changes, select the changes to commit, right-click, and select "stage selected ranges". Git commits only the changes that are "staged". You can also un-stage changes by selecting and right-clicking them.

If it's easier, you can also revert only parts of a document in the diff view by selecting the part to be reverted, right clicking, and choosing "revert selected ranges".

Reply:
My point was, that we should check why prettier is doing this.
Because the next time we (or anyone else) touches this file, they will run into the same issue. Currently, it doesn't seem possible with doctoc (see https://github.com/thlorenz/doctoc/issues/213) to ignore headings though, so I don't think there is another way of solving this for now. 

Done. PTAL.

Reply:
I don't think Prettier is doing this since Prettier never changes content like this (it only formats it) and TOC are not even an official part of the Markdown spec. This is done by a much more invasive tool. I suspect a VSCode plugin like https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one here. That's at least the culprit in my setup. Super useful plugin btw.

Reply:
Ohh, right! That's true. I have it installed too...

Well, then we could add `<!-- omit in toc -->` (see [1] from their README) to the end of the heading `# Contribute to Ory $PROJECT`. I am not sure if that'll break anything, though. I tested it and at least it doesn't generate the heading into the TOC anymore. WDYT?

1:
![image](https://user-images.githubusercontent.com/5540255/191958380-fd60cf30-985a-44de-afa5-4457f63dd798.png)

Reply:
Sounds promising. Since that's a separate issue and this branch is shipped already, do you want to give it a try in a separate feature branch?

Reply:
Sure thing: https://github.com/ory/meta/pull/192